### PR TITLE
Install the wheel

### PR DESCRIPTION
**Pull request by sbrunner**
To be sure that we didn't miss any file in the package

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -31,7 +31,8 @@ jobs:
           patterns: pypi docker
 
       - run: echo "${HOME}/.local/bin" >> ${GITHUB_PATH}
-      - run: python3 -m pip install --user --editable=.
+      - run: python3 ./setup.py egg_info --no-date bdist_wheel
```
Comment (arnaud-morvan) [inline .github/workflows/main.yaml:34]: We do not need `- run: python3 -m pip install --user wheel` here ?

File: .github/workflows/rebuild.yaml [hunk 2]
```diff
@@ -30,7 +30,8 @@ jobs:
           patterns: pypi docker
 
       - run: echo "${HOME}/.local/bin" >> ${GITHUB_PATH}
-      - run: python3 -m pip install --user --editable=.
+      - run: python3 ./setup.py egg_info --no-date bdist_wheel
```
Comment (arnaud-morvan) [inline .github/workflows/rebuild.yaml:33]: We do not need `- run: python3 -m pip install --user wheel` here ?

**Review by arnaud-morvan — APPROVED** [review]
LGTM, see coments

**Comment by sbrunner** [comment]
No, it's not needed, in the audit it's needed because of asdf-vm...

**Review by arnaud-morvan — APPROVED** [review]
Ok, thanks

**Pull request merged**